contributors/design-proposals/node/kubelet-cri-logging.md

```diff
@@ -133,10 +133,10 @@ PodSandboxConfig.LogDirectory: /var/log/pods/<podUID>/
 ContainerConfig.LogPath: <containerName>_<instance#>.log
 ```
 
-Because kubelet determines where the logs are stores and can access them
-directly, this meets requirement (1). As for requirement (2), the log collector
+Because kubelet determines where the logs are stored and can access them
```

Comment:
Since kubelet knows that where is the log of each pod, so this is more suitable for requirement 2: "Allow kubelet to manage the lifecycle of the logs to pave the way for better disk management in the future."

Since log collector can easily extract basic pod metadata, so this is more suitable for requirement 3: "Allow log collectors to easily integrate with Kubernetes across different container runtimes while preserving efficient storage and retrieval."